hack/run-operator-catalog.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+# hack/run-operator-catalog.sh
+#
+# Run the operator from a catalog image.
+# Required environment variables:
+#
+# - CATALOG_IMG: catalog image to deploy
+# - CSV_VERSION: version tag of the cluster service version
+#
+# Optional environment variables
+#
+# - KUSTOMIZE_BIN: path to kustomize
+# - KUBECTL_BIN: path to kubectl (or equivalent command line)
+# - SED_BIN: path to GNU sed
+# - CATALOG_NAMESPACE: Namespace to deploy the catalog. Defaults to shipwright-operator.
+# - SUBSCRIPTION_NAMESPACE: Namespace to install the operator via an OLM subscription. Defaults to
+#   shipwright-operator.
+# - NAME_PREFIX: prefix to use for all resource names. Defaults to "shipwright-"
+
+set -e
+
+kustomize="${KUSTOMIZE_BIN:-${PWD}/bin/kustomize}"
```

Comment:
nit: global Bash script variables are, by convention, capitalized and scoped variables (`local`) are lowercase. For simplicity we can reuse the default variable name, like for instance `KUSTOMIZE_BIN` straightaway instead of a arbitrary rename.